java/code_to_gast/java_router.py

```diff
@@ -30,6 +31,11 @@ def node_to_gast(node):
         return node_to_gast(node.expression)
     elif type(node) == list:
         return java_helpers.node_list_to_gast_list(node)
+    elif type(node) == javalang.tree.LocalVariableDeclaration:
+        # our current gAST doesn't support multiple declarations/unsure what that is
```

Comment:
I don't have any fixes to suggest, but this seems like it might lead to silent failures.

Reply:
https://stackoverflow.com/questions/6202818/initializing-multiple-variables-to-the-same-value-in-java#:~:text=You%20can%20declare%20multiple%20variables,opposite%20of%20%22visually%20simple%22.

Maybe detect that case and return feature not supported error?

Reply:
Maybe also file a ticket for js and python since they both support though with different syntaxs.